dapps/marketplace/src/utils/pasteIntoInput.js

```diff
@@ -0,0 +1,17 @@
+export default function pasteIntoInput(el, text) {
```

Comment:
nit: could you document what this new method does using JSDoc style comments ?

Reply:
done